packages/preview/optimal-ovgu-thesis/0.1.0/README.md

```diff
@@ -0,0 +1,59 @@
+This template was created for a master thesis at the faculty of computer science (FIN), but should work as well for other faculties.
+
+## File structure
+```
+.
+├── assets                          // Images, CSV-Files, etc. 
+│   └── figure                      // Image files
+│       └── optimal-ovgu-thesis    
+├── chapter                         // Content
+│   ├── 01-Einleitung.typ
+│   ├── ...
+│   └── 99-Appendix.typ
+├── expose.typ                      // Exposé template
+├── metadata.typ                    // Metadata and template config
+├── thesis.bib                      // Bibliography (e.g. generated by Zotero + Better BibTex)
+└── thesis.typ                      // Thesis template
+```
+
+## Logos on the title page

```

Comment:
Are the logo assets intentionally part of the template that will be cloned into the user's project rather than part of the package, so that they can easily be swapped? If they are always the same, it might be user-friendlier to just ship them behind the scenes.

Reply:
Yes, that was intentional for exact this reason to make it easier for users to swap them out. But I am unsure if it's correct license wise or if it'd be better to just provide links for the users to download them themselves.

Reply:
Just contacted the office at ovgu and asked for the license permission

Reply:
Okay, let me know once this is ready for merging.

Reply:
I removed the logo assets with questionable licenses. This should be ready now @laurmaedje . Thanks for reviewing it!